torch/_inductor/fx_passes/fuse_attention.py

```diff
@@ -546,16 +555,26 @@ def _get_sfdp_patterns():
         torch.empty, (1024, 128, 128), device=device, requires_grad=True
     )
 
+    # when batch_size = 1, some clones disappear due to no memory layout change
```

Comment:
Can we describe the graph difference (w/wo clone) between bs1 and bsN in more details?

Reply:
Thanks and modified！